src/components/AssetSearch/AssetList.tsx

```diff
@@ -23,17 +24,69 @@ type ItemData<T> = {
   handleClick: T
 }
 
+const sortAssetsByAccountAndMarketCap = (
+  assets: Asset[],
+  enrichedData: Record<string, { fiatAmount: string; cryptoAmount: string; marketCap: string }>,
+): Asset[] => {
+  return sortBy(assets, [
+    asset => {
+      return Number(enrichedData[asset.assetId].fiatAmount)
+    },
+    asset => {
+      return Number(enrichedData[asset.assetId].marketCap)
+    },
+  ]).reverse()
+}
+
+/**
+ * This function create a data object(cryptoAmount, fiatAmount, marketCap) from the Asset, portfolioAcountRows and marketData for each asset for facilitate sorting.
+ * @param assets
+ * @param portfolioAcountRows
+ * @param marketData
+ * @returns Record<string, { fiatAmount: string; cryptoAmount: string; marketCap: string }>
+ */
+export const createEnrichData = (
```

Comment:
Same comment about enrich the verb vs. enriched the adjective, it might seem like a nitpick but this is an open-source project and we should keep such terminology as clean as possible.